figures/pipeline/site_daily_metrics.py

```diff
@@ -88,6 +88,32 @@ def get_previous_cumulative_active_user_count(site, date_for):
         return 0
 
 
+def get_site_active_learners_for_date(site, date_for):
+    """
+    Get the active learners ids for the given site and date
+
+    We do this by filtering StudentModule for courses in the site, then
+    for StudentModule records filtered for the date, then we get the distinct
+    learners ids
+    """
+    student_modules = get_student_modules_for_site(site)
```

Comment:
@asadmanzoor93 Can we make use of our MAU's model to extract this information?
@rehan99000 your thoughts on this?

Reply:
@zubair-arbi figures provide stats of active users on basis of activity in any course. This is their constant logic throughout the APIs. If we want to replace this then we will eventually end up replacing more of the APIs for similar logic.

Reply:
@zubair-arbi I assume you are talking about the logic of this whole method. I think we can. we have the side as well as the date in that model.

Reply:
@zubair-arbi as per our discussion, we will keep this as it is and update the tooltip text to reflect the correct logic.